### PR TITLE
Setup Folder Structure and Gulp

### PR DESCRIPTION
**Pull request by william-abboud**
Set up folder structure
Setup gulp tasks
Setup JSHint
Setup Examples

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,10 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <title>S.W.E.A.T</title>
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <p>Hello World</p>
```
Comment (moonclash) [inline src/index.html:9]: Project has been set up and tested. Dummy content no longer needed, please delete code in lines 9 -12

File: src/index.html [hunk 2]
```diff
@@ -3,10 +3,14 @@
...
     <p>hello em</p>
```
Comment (embrah) [inline src/index.html]: Project has been set up and tested. Dummy content no longer needed, please delete code in lines 9 -12

File: src/scripts/main.js [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log('pesho');
```
Comment (moonclash) [inline src/scripts/main.js]: Have you tested this? Is the script working correctly? If you did, please delete line 1.

File: src/styles/main.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+body {
+  background: blue;
```
Comment (moonclash) [inline src/styles/main.scss]: Remove background, unless you plan to use it for the theme.

File: src/index.html (hunk 1 above)
Comment (william-abboud) [inline src/index.html:9]: I will keep the Hello World Paragraph just so there is something on the page before we decide on a structure :) :+1:

File: src/scripts/main.js (hunk 3 above)
Comment (william-abboud) [inline src/scripts/main.js]: Yes it is, However I will throw in a hello world example just for now so the JS spits out anything and if an error occurs we know about it

File: src/styles/main.scss (hunk 4 above)
Comment (william-abboud) [inline src/styles/main.scss]: Good shout, will do :+1:

**Comment by moonclash** [comment]
:shipit:

**Pull request merged**